CHANGELOG.asciidoc

```diff
@@ -45,6 +45,7 @@ https://github.com/elastic/beats/compare/v6.4.0...master[Check the HEAD diff]
 - Add backoff support to x-pack monitoring outputs. {issue}7966[7966]
 - Fix a race condition with the `add_host_metadata` and the event serialization. {pull}8223[8223]
 - Enforce that data used by k8s or docker doesn't use any reference. {pull}8240[8240]
+- Switch to different UUID lib: github.com/gofrs/uuid. {pull}8485[8485]
```

Comment:
Can we insert why we switch probably adding a word about collision in the old library.